packages/suite/src/utils/suite/homescreen.ts

```diff
@@ -3,9 +3,12 @@ import { DeviceModelInternal } from '@trezor/connect';
 import { deflateRaw } from 'pako';
 
 export const deviceModelInformation = {
```

Comment:
Would be grate to add here `: Record<DeviceModelInternal, {....}>` typings so when adding new device typescript tells us